app/tests/spec/lib/experiment.js

```diff
@@ -70,6 +70,12 @@ define(function (require, exports, module) {
         assert.equal(expInt3.forceExperiment, 'something');
         assert.isTrue(expInt3.initialized, 'forceExperiment overrides user agent');
       });
+
+      it('supports webdriver override', () => {
+        expOptions.window.navigator.webdriver = true;
+        const expInt4 = new ExperimentInterface(expOptions);
+        assert.isFalse(expInt4.initialized, 'do not init if webdriver override');
+      });
```

Comment:
Is it worth adding a "forceExperiment overrides webdriver" test similar to the "forceExperiment overrides user agent" one above?

Reply:
Sure!